src/main/kotlin/step2/StringCalculator.kt

```diff
@@ -0,0 +1,35 @@
+package step2
+
+private const val INPUT_NULL_ERROR_MESSAGE = "입력 값은 null일 수 없습니다."
+
+private const val INPUT_BLANK_ERROR_MESSAGE = "입력 값은 공백일 수 없습니다."
+
+private const val OPERATOR_ERROR_MESSAGE = "올바른 사칙연산 기호를 입력해주세요."
+
+private const val DELIMITER = " "
+
+class StringCalculator {
+
+    fun calculate(input: String?): Int {
+        val parseList = parse(input)
+        var result = parseList[0].toInt()
+        for (i in 1 until parseList.size step (2)) {
```

Comment:
사소하지만 step 뒤의 2를 소괄호로 감쌀 필요가 없을 것 같아요. 😃
```suggestion
        for (i in 1 until parseList.size step 2) {
```


추가적으로 전체적으로 매직넘버에 대해서 잘 처리해주셨는데, 이런 경우도 매직넘버로 볼 수 있지 않을까 싶어요. 🤔
다른 케이스와 동일하게 매직넘버를 상수로 추출해보면 어떨까요?

Reply:
수정하겠습니다!